test/test_parabolic_3d.jl

```diff
@@ -0,0 +1,88 @@
+module TestExamplesParabolic3D
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# EXAMPLES_DIR = joinpath(examples_dir(), "dgmulti_3d")
+
+# Start with a clean environment: remove Trixi output directory if it exists
+outdir = "out"
+isdir(outdir) && rm(outdir, recursive=true)
+
+@testset "SemidiscretizationHyperbolicParabolic" begin
```

Comment:
```suggestion
@testset "SemidiscretizationHyperbolicParabolic (3D)" begin
```
To make it clear that this is the 3D part. If the 2D part is missing the same annotation, it would be great if we can add it there as well!

Reply:
I added such an annotation to the 2D test file as well.